packages/docs/site/docs/blueprints/09-troubleshoot-and-debug-blueprints.md

```diff
@@ -10,7 +10,53 @@ When you build Blueprints, you might run into issues. Here are tips and tools to
 ## Review Common gotchas
 
 -   Require `wp-load`: to run a WordPress PHP function using the `runPHP` step, you’d need to require [wp-load.php](https://github.com/WordPress/WordPress/blob/master/wp-load.php). So, the value of the `code` key should start with `"<?php require_once('wordpress/wp-load.php'); REST_OF_YOUR_CODE"`.
--   Enable `networking`: to access wp.org assets (themes, plugins, blocks, or patterns), or load a stylesheet using [add_editor_style()](https://developer.wordpress.org/reference/functions/add_editor_style/) (say, when [creating a custom block style](https://developer.wordpress.org/news/2023/02/creating-custom-block-styles-in-wordpress-themes)), you’d need to enable the `networking` option: `"features": {"networking": true}`.
+
+## Common Issues and Solutions
+
+### WP-CLI: Error Establishing a Database Connection on Mounted Sites
+
+When using `wp-cli` with a mounted Playground site (e.g., via `--mount-before-install`), you might encounter an "Error establishing a database connection." This happens because WordPress Playground loads the SQLite database integration plugin from its internal files by default, not from the mounted directory, meaning it's not persisted for external `wp-cli` calls.
+
+To resolve this, you need to explicitly install and configure the SQLite database integration plugin within your Blueprint.
+
+**Solution:** Add the following steps to your Blueprint:
+
+```json
+{
+	"steps": [
+		{
+			"step": "installPlugin",
+			"pluginData": {
+				"resource": "wordpress.org/plugins",
+				"slug": "sqlite-database-integration"
+			}
+		},
+		{
+			"step": "cp",
+			"fromPath": "/wordpress/wp-content/plugins/sqlite-database-integration/db.copy",
+			"toPath": "/wordpress/wp-content/db.php"
+		},
+		{
+			"step": "activatePlugin",
+			"pluginPath": "/wordpress/wp-content/plugins/sqlite-database-integration"
+		}
+	]
```

Comment:
Would this shorthand form also do the trick? AFAIR it installs and activates the plugin – I wonder if there's any activation hook that handles copying the `db.php` file?


```suggestion
	"plugins": [ "sqlite-database-integration" ]
```